logic/src/main/nl/jft/logic/statistic/AbstractStatistic.java

```diff
@@ -0,0 +1,8 @@
+package nl.jft.logic.statistic;
+
+/**
+ * @author Lesley
+ */
+public abstract class AbstractStatistic implements Statistic {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Convert the abstract class "AbstractStatistic" into an interface. (sonar.java.source not set. Assuming 8 or greater.) [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://92.110.24.119:9000/coding_rules#rule_key=squid%3AS1610)